Lesson24/deniz/newsletter assignment/index.js

```diff
@@ -0,0 +1,9 @@
+const form = document.querySelector('form');
+const card_1 = document.querySelector('.card');
+const card_2 = document.querySelector('.card-2');
+
+form.addEventListener('submit', function(e) {
+    e.preventDefault();
```

Comment:
Here, you also need to do the actual validation in JS to check if the input is actually an email address. You can use a regex that we used in the lesson to complete this assignment.

Reply:
shall I re-write this part ? so i need to remove card_1 and card_2 right?

Reply:
```
    card_1.classList.add('hide');
    card_2.classList.remove('hide');
```

This part is correct, I don't think you need to change it. But you need to add code to validate the input in between `e.preventDefault();` and `card_1.classList.add('hide');
    card_2.classList.remove('hide');`